### ISSUE TITLE
Cloudflare blocking user-agent

### ISSUE THREAD

**Issue by Loki919** [ok24601/etoro-api#44]
while calling getAccountData the response is "error 1020". You can avoid this by changing the user-agent to 
"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.212 Safari/537.36"
In RequestUtils.prepareRequest(..). This worked for me. The error itselfs seems to be thrown by cloudflare.

Greetings

**Comment by pedroestima-sgg** [comment]
It didn't help on my end. During the last 3 days I managed to make it work 3 or 4 times just be keep trying without any changes, looks like sometimes it connects to a server that doesn't throw that error

**Comment by Loki919** [comment]
I work together with a friend with this api. He had the same problem at first.  After updating his Chrome browser it worked for him to!
I use  Version 90.0.4430.212 (Offizieller Build) (64-Bit)
We also use some updated version of the chromedriver. Chromedriver V87x does not work with chrome 90.x

**Comment by pedroestima-sgg** [comment]
I don't think it is related to the browser since it's just doing a request using a HttpClient, unless the browser version could change the type of cookies being returned when it first logs in?

**Comment by pedroestima-sgg** [comment]
But indeed, it looks like CloudFlare is making some kind of check on browser integrity. I tried with the `user-agent` from my logged in session in the browser and it worked (at least for now)
